docs/offroadvoertuigen.md

```diff
@@ -0,0 +1,62 @@
+# Offroad Voertuigen lijst
+
+De Lijst hieronder geeft aan welke voertuigen vanuit de gemeente zijn toegestaan om te gebruiken buiten de geasfalteerde wegen, we klassifiseren dit onder:
+| Classificering | Uitleg | Omschrijving |
+|-|---|---|
+|V| Verharde Baan    | een verhard weg waarbij het wegdek uit asfalt of beton bestaat |
+|O| Onverharde Baan  | een onverhard pad waarbij het wegdek uit zand bestaat |
+|A| Off-Road         | een pad waarbij er geen wegdek is en de ondergrond uit gras, zand of modder bestaat |
+
+## Overtreding
+
+Bij het rijden met een voertuig op een pad of weg waarvoor deze niet bedoeld is valt dit onder "Artikel 10 - Deelname aan het verkeer"

```

Comment:
Bij het rijden met een voertuig op een pad of weg, waarvoor deze niet bedoeld is valt dit onder "Artikel 10 - Deelname aan het verkeer" van de APV